nifi-nar-bundles/nifi-sql-reporting-bundle/nifi-sql-reporting-tasks/pom.xml

```diff
@@ -81,6 +81,11 @@
             <version>1.11.0-SNAPSHOT</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-rules-engine-service-api</artifactId>
```

Comment:
I think this can be of `provided` scope, as we get it from the `nifi-standard-services-api-nar`.

Reply:
makes sense